README.md

```diff
@@ -16,7 +16,9 @@ npm install @adobe/node-fetch-retry
 
 ## Usage
 
-This library works the same as the normal [`fetch api`](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API), but with some added features.
+This library works the same as the normal [`fetch api`](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API), but with some added features. 
+
+For instance, you can set a `headers` object in the `options` used to make the HTTP(S) request.
```

Comment:
These 2 sentences together sound like the `headers` is a _new_ feature of node-fetch-retry.

We should say that it is the normal fetch API and the options works as is, with the addition of the `retryOptions` child object. Then we don't have to mention `headers` or any other existing option explicitly.